crypto/src/lokinet_identity.cpp

```diff
@@ -56,10 +54,11 @@ std::vector<uint8_t> parseLokinetIdentityPublic(const std::string& path) {
             "Lokinet identity.private file could not be found");
     }
     std::ifstream input(p.c_str(), std::ios::binary);
-    const std::vector<uint8_t> bytes(std::istreambuf_iterator<char>(input), {});
-    const std::vector<uint8_t> publicKey(bytes.begin() + publicKeyOffset,
-                                         bytes.begin() + publicKeyOffset +
-                                             keyLength);
+    const std::vector<uint8_t> privateKey(std::istreambuf_iterator<char>(input), {});
```

Comment:
Probably best to update the functions name, since we don't read the lokinet indentity.private file:
parseLokinetIdentityPublic
parseLokinetIdentityPrivate

Also, since the public key is calculated from the private key, there's no need to read the same file twice...